backend/account/migrations/0001_initial.py

```diff
@@ -0,0 +1,27 @@
+# Generated by Django 5.0.1 on 2024-02-01 21:39
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='UserRegister',
+            fields=[
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('username', models.CharField(max_length=30)),
+                ('email', models.EmailField(max_length=50)),
+                ('dateofbirth', models.DateTimeField()),
+                ('gender', models.CharField(choices=[('M', 'Male'), ('F', 'Female'), ('O', 'Other')], max_length=1)),
+                ('password', models.CharField(max_length=128)),
+                ('confirm_password', models.CharField(max_length=128)),
+                ('created_at', models.DateTimeField(auto_now_add=True)),
+            ],
+        ),
+    ]
```

Comment:
This migration file was created from UserRegister which was already deleted and used no more, so might be good idea to delete it as well.
`backend/account/migrations/0001_initial.py`
`backend/account/migrations/0002_delete_userregister.py`

After deleting it, run `makemigrations` command and commit the new changes.

Reply:
done